Vic2ToHoI4/Source/Mappers/CountryMapping.cpp

```diff
@@ -97,34 +74,39 @@ void CountryMapper::resetMappingData()
 }
 
 
-void CountryMapper::makeOneMapping(const std::string& Vic2Tag)
+void CountryMapper::makeOneMapping(const std::string& Vic2Tag, bool debug)
 {
 	auto mappingRule = Vic2TagToHoI4TagsRules.find(Vic2Tag);
 
 	bool mapped = false;
 	if (mappingRule != Vic2TagToHoI4TagsRules.end())
 	{
 		auto possibleHoI4Tags = mappingRule->second;
-		mapped = mapToFirstUnusedVic2Tag(possibleHoI4Tags, Vic2Tag);
+		mapped = mapToFirstUnusedVic2Tag(possibleHoI4Tags, Vic2Tag, debug);
 	}
 
 	if (!mapped)
 	{
 		std::string HoI4Tag = generateNewHoI4Tag(Vic2Tag);
-		mapToNewTag(Vic2Tag, HoI4Tag);
+		mapToNewTag(Vic2Tag, HoI4Tag, debug);
 	}
 }
 
 
-bool CountryMapper::mapToFirstUnusedVic2Tag(const std::vector<std::string>& possibleHoI4Tags, const std::string& Vic2Tag)
+bool CountryMapper::mapToFirstUnusedVic2Tag(const std::vector<std::string>& possibleHoI4Tags,
+	 const std::string& Vic2Tag,
+	 bool debug)
 {
 	for (auto possibleHoI4Tag: possibleHoI4Tags)
 	{
 		if (!tagIsAlreadyAssigned(possibleHoI4Tag))
 		{
 			V2TagToHoI4TagMap.insert(make_pair(Vic2Tag, possibleHoI4Tag));
 			HoI4TagToV2TagMap.insert(make_pair(possibleHoI4Tag, Vic2Tag));
-			logMapping(Vic2Tag, possibleHoI4Tag, "mapping rule");
+			if (debug)
```

Comment:
yknow, we could do this the smart way and have Log(LogLevel::Debug) not do anything unless debug was stated globally, as in Log.setDebug().

Reply:
Yep, that's on my list for future cleanup. I should formally put it on the list, though.